airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -152,6 +153,13 @@ def _get_authenticator(config: Dict[str, Any]):
             creds = config.get("credentials")
             token = creds.get("access_token") or creds.get("personal_access_token")
         tokens = [t.strip() for t in token.split(TOKEN_SEPARATOR)]
+        requests_per_hour = config.get("requests_per_hour")
+        if requests_per_hour:
+            return MultipleTokenAuthenticatorWithRateLimiter(
```

Comment:
Let's add unit tests for `MultipleTokenAuthenticatorWithRateLimiter` usage.

Reply:
I have added unit_test for MultipleTokenAuthenticatorWithRateLimiter class